lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -171,7 +175,7 @@ def __repr__(self):
     @classmethod
     def from_preloaded(
```

Comment:
FYI changes in this method are not pertinent to the PR, but I found they were useful in my development process.